master/pkg/tasks/task.go

```diff
@@ -83,6 +83,28 @@ func ToContainerSpec(t TaskSpec) container.Spec {
 		shmSize = t.TaskContainerDefaults.ShmSizeBytes
 	}
 
+	var capAdd []string
+	if env.AddCapabilities != nil {
+		capAdd = *env.AddCapabilities
+	}
```

Comment:
Said this earlier but I think `env.AddCapabilities` can just be a `[]string` and directly placed here (in both cases `capAdd` would end up being a nil string slice).